app/controllers/api/v1/packages_controller.rb

```diff
@@ -14,7 +16,7 @@ class PackagesController < Api::V1::ApiController
 
     def_param_group :package do
       param :package, Hash, required: true do
-        param :quantity, Integer, desc: "Package quantity", allow_nil: true
+        param :quantity, lambda { |val| [String, Fixnum].include? val.class }, desc: "Package quantity", allow_nil: true
```

Comment:
@steveyken 

when we receive request from gc-admin 

    Started POST "/api/v1/packages" for 122.169.90.136 at 2016-01-14 11:52:09 +0530
    Processing by Api::V1::PackagesController#create as JSON
    Parameters: {"package"=>{"quantity"=>1, "length"=>6, "width"=>54, "height"=>89, "notes"=>"Test item", "state"=>"expecting", "state_event"=>nil, "received_at"=>nil, "rejected_at"=>nil, "created_at"=>nil, "updated_at"=>nil, "image_id"=>nil, "offer_id"=>519, "inventory_number"=>nil, "item_id"=>"811", "package_type_id"=>"99", "location_id"=>nil}}

When we recieve request from stockit

    Started POST "/api/v1/packages.json" for 127.0.0.1 at 2016-01-14 11:54:02 +0530
    Processing by Api::V1::PackagesController#create as JSON
    Parameters: {"package"=>{"designation_name"=>"Lapsap - China - Hong Kong (Special Administrative Region)", "item"=>{"donor_condition_id"=>"Lightly Used"}, "notes"=>"kk kkk kkkk", "inventory_number"=>"E55668", "location_id"=>"214", "quantity"=>"4"}}

here, the quantity field value is in string and fixnum format.